packages/docs/src/components/layout/SideNav/SideNav.tsx

```diff
@@ -106,9 +106,9 @@ const SideNav = ({ location }: SideNavProps) => {
       })}
     >
       <UsaBanner className="ds-u-display--block ds-u-md-display--none" />
-      <header className="c-navigation__header ds-u-md-display--block ds-u-md-display--none">
```

Comment:
removing the `ds-u-md-display--block` class as the following `ds-u-md-display--none` cancels it out.